lib/rheo/Makefile.lammps

```diff
@@ -0,0 +1,7 @@
+# Settings that the LAMMPS build will import when this package is installed
```

Comment:
This is technically correct, but please it could be more automated using pkg-config or pkgconf.
Check out lib/kim, lib/kokkos, lib/qmmm

Reply:
I don't disagree with that! I did look at some of those libraries, but their automated solutions were out of my wheelhouse. I can take a stab at something more sophisticated, but it won't be for a while. I'll have to carve out time to work through some tutorials.

Reply:
I can do it. I've done far more complex stuff in src/Makefile ... :wink:

Reply:
Please have a look at my tweaks to the package. I also did some spell-checking...

Reply:
That's really helpful and works great on my system. Thanks! I'll look through the changes in more detail and parse all the new additions.